conda-environment.yml

```
@@ -5,7 +5,7 @@ dependencies:
   - ansimarkup >=1.0.0
   - async-timeout>=3.0.0 # [py<3.11]
   - colorama >=0.4,<1.0
-  - graphene >=2.1,<3
+  - graphene ==3.4.3
```

Can we loosen this:
* We need to support newer versions (e.g. 3.4.4)
* We need to support Python 3.7, the oldest version of graphene that supports Python3.7 on conda-forge is 3.4.0.

(to test compatibility of conda-forge packages, try this `mamba create -n test  python=3.7 graphene=3.4.0`)

```suggestion
  - graphene >=3.4.0
```

---

We can't upgrade major and minor versions, because we need to avoid a `graphene` version that adopts a new version of `graphql-core` (because HEAD is not compatible with our `subscribe.py` workaround) ..
I think the best we can do is `graphene >=3.4.0,<3.5`

---

Fixed..